<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgTextOptions</class>
 <widget class="QWidget" name="WdgTextOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="cmbStyle">
       <item>
        <property name="text">
         <string>Foreground Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Background Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gradient</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelMode">
       <property name="text">
        <string>Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelStyle">
       <property name="text">
        <string>Style:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonArtistic">
       <property name="text">
        <string>Artistic</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonMultiline">
       <property name="text">
        <string>Multiline</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="SpecialSpacer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
